### ISSUE TITLE
breakoutRequest missing from php-cgi-wasm

### ISSUE THREAD

**Issue by mglaman** [seanmorris/php-wasm#42]
`PhpCgiBase.mjs` has `import { breakoutRequest } from './breakoutRequest.mjs';`, which fails on a 404 since it's not bundled in `php-cgi-wasm` when using a worker in module mode.

index.html
```
            const serviceWorker = navigator.serviceWorker;
            serviceWorker.register(`/worker.mjs`, {
                type: "module"
            });
```

worker.mjs

```
import { PhpCgiWorker } from "./PhpCgiWorker.mjs";

const php = new PhpCgiWorker({
  onRequest,
  notFound,
  sharedLibs,
  prefix: "/cgi/",
  docroot: "/persist/www",
  types: {
    jpeg: "image/jpeg",
    jpg: "image/jpeg",
    gif: "image/gif",
    png: "image/png",
    svg: "image/svg+xml",
  },
});
```

**Comment by mglaman** [comment]
Same with parseResponse.mjs

Maybe it's because I'm not using a bundler.

**Comment by seanmorris** [comment]
This has been fixed in `php-cgi-wasm@0.0.9-alpha-2`

https://www.npmjs.com/package/php-cgi-wasm/v/0.0.9-alpha-2?activeTab=code

**Comment by mglaman** [comment]
Right on, thanks!
